project/plugins.sbt

```diff
@@ -16,8 +16,8 @@ addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.8.1")
 
 addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.3")
 
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.32")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.6.0")
```

Comment:
Can we use a minimal backward compatible version, like 1.1.0 here?

The reason is that if we use the latest one, then we **force** our library users to also upgrade to the latest Scala.js version as well in order to use quill.

Reply:
Good point. Thanks. I'll drop the version to 1.1.1, which is compatible with 1.1.0:
http://www.scala-js.org/news/2020/07/02/announcing-scalajs-1.1.1/

And I hope this PR will be reviewed soon. I'm waiting on this to update other libraries that have dependency on Quill sjs module :)

Reply:
Thanks @ithinkicancode for adjusting it!

I'm not a maintainer, so cannot tell how long it will take to merge it, but I'm also looking forward to it!

Maybe, one thing that would potentially speed up the merge, I think, is there a way to keep the cross compile for 2.11? Or its not possible any more?

Reply:
Yeah, I'd like to keep it too but BooPickle doesn't have a version that supports both ScalaJs 1.x and Scala 2.11.

Reply:
@viktor-podzigun So unfortunately I had to bump scalajs to 1.6 because, while rebasing with master, master's 2.12 version has been updated to 2.12.14. There is no older scalajs plugin artifact for 2.12.14 but 1.6.0.